src/bundle/Resources/public/js/scripts/admin.anchor.sections.navigation.js

```diff
@@ -15,11 +16,19 @@
         return [...secondarySectionItems];
     };
     let currentlyVisibleSections = getSecondarySectionActiveItems();
+    const getFirstSection = (sectionGroup) => {
+        return sectionGroup.querySelector('.ibexa-edit-content__secondary-section');
+    };
+    const getLastSection = (sectionGroup) => {
+        const sections = sectionGroup.querySelectorAll('.ibexa-edit-content__secondary-section');
+
+        return sections ? [...sections].at(-1) : null;
```

Comment:
`sections` will always be true
```suggestion
        return [...sections].at(-1);
```

Reply:
also, I think it is too early for `.at`, unfortunately. 😞 
https://caniuse.com/mdn-javascript_builtins_array_at